### PR TITLE
Add many corrections to dictionary.txt

### PR DESCRIPTION
**Pull request by skangas**
This adds several typos that I have in my own personal list of auto-corrections. I also added in several typos found in recent months in Emacs.

In other words, all these typos have been made in the wild at least once, often by yours truly.

### CODE REVIEW

**Comment by luzpaz** [comment]
@skangas TY for the PR. Do you mind doing a 2nd run over your PR and add some plural forms that are missing?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4131,6 +4141,7 @@ backwad->backwards
 backwardss->backwards
 backware->backward
 backwark->backward
+backwars->backwards
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:4144]: can also be `backward` since `d` key is right of `s`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5350,6 +5364,7 @@ cantact->contact
 cantacted->contacted
 cantacting->contacting
 cantacts->contacts
+cantain->contain
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:5367]: add `cantains->contains`

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5789,6 +5808,7 @@ channael->channel
 channe->channel
 channeles->channels
 channes->channels, chances, changes,
+channge->change
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:5811]: Add `channges->changes`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -7127,6 +7152,7 @@ compicated->complicated
 compications->complications
 compied->compiled
 compilability->compatibility
+compilaiton->compilation
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:7155]: Add `compilaitons->compilations`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -8401,6 +8429,7 @@ contstructing->constructing
 contstruction->construction
 contstructor->constructor
 contstructors->constructors
+conttributing->contributing
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:8432]:
```
conttribute->contribute
conttributes->contributes
conttributed->contributed
conttribution->contribution
conttributions->contributions
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -8638,6 +8667,7 @@ copyeight->copyright
 copyeighted->copyrighted
 copyeights->copyrights
 copyied->copied
+copyirght->copyright
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:8670]: `copyirghts->copyrights`

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -11122,6 +11157,7 @@ dialgos->dialogs
 dialig->dialog
 dialigs->dialogs
 dialoge->dialog, dialogue,
+dialouge->dialogue
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:11160]: `dialouges->dialogues`

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -24368,6 +24451,7 @@ packges->packages
 packgs->packages
 packhage->package
 packhages->packages
+packkage->package
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:24454]: `packkages->packages`

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -37206,19 +37326,22 @@ worarounds->workarounds
...
 worflows->workflows
+worht->worth
 workaorund->workaround
 workaorunds->workarounds
 workaound->workaround
 workaounds->workarounds
 workaraound->workaround
 workaraounds->workarounds
 workarbound->workaround
+workaronud->workaround
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:37344]: `workaronuds->workarounds`

**Review by luzpaz — CHANGES_REQUESTED** [review]

**Comment by skangas** [comment]
> @skangas TY for the PR. Do you mind doing a 2nd run over your PR and add some plural forms that are missing?

Sure! Please see the latest commit with my fixes for the issues you mentioned. Thanks for reviewing.

BTW, since you mentioned key placement: note that I use dvorak instead of qwerty. So my typos will be a bit different.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**